### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by kanna-1**

### CODE REVIEW

File: docs/team/kanna-1.md [hunk 1]
```diff
@@ -10,39 +10,45 @@ HealthSync is an application designed to streamline the workflow of healthcare c
...
+* **Project management**:
+    * Updated the application name ([#80](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/80))
+    * Updated the application icon ([#126](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/126))
+    * Refactored entire code base to remove all traces of the inherited `Tags` field ([#198](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/198))
+
 
 * **Documentation**:
     * User Guide:
         * Added documentation for the features `edit`
         * Added dcoumentation for the features `undo`
```
Comment (WZWren) [inline docs/team/kanna-1.md:37]: You missed a typo here.

File: docs/team/kanna-1.md [hunk 2]
```diff
@@ -10,39 +10,45 @@ HealthSync is an application designed to streamline the workflow of healthcare c
 Given below are my contributions to the project.
 
 * **New Feature**: Added the ability to undo a Command
-  * What it does: allows the user to undo an edit, add and delete command
-  * Justification: This feature allows users to be able to quickly recover from errors made without having to go through a series of other commands to achieve the same goal.
+    * What it does: allows the user to undo an edit, add, delete and clear command.
+    * Justification: This feature allows users to be able to quickly recover from errors made without having to go through a series of other commands to achieve the same goal.
+    * Implemention: A new abstract class undo-able command was implemented to do this.
```
Comment (WZWren) [inline docs/team/kanna-1.md:15]: Typo in `Implementation`.

**Review by WZWren — COMMENTED** [review]
Hey, you've made some typos. It should be ready to merge immediately after fixing, so feel free to merge once you fix the typos.

**Comment by kanna-1** [comment]
Fixed the typos. Merging

**Pull request merged**